gprofiler/main.py

```diff
@@ -645,11 +600,11 @@ def main():
             remote_logs_handler.init_api_client(client)
 
         runtimes = {
-            "system": args.perf_mode != "none",
-            "java": args.java,
-            "python": args.python_mode != "none",
-            "php": args.php,
-            "ruby": args.ruby,
+            "system": args.perf_mode not in ["none", "disabled"],
+            "java": args.java and args.java_mode == "enabled",
```

Comment:
Why do you need to check 2 fields? What sense does it make to have 2 fields o.O can make `--no-java` write `disabled` to `java_mode`.